modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -48,6 +49,21 @@ export class Utils implements BaseUtils {
     throw new NotImplementedError('method not implemented');
   }
 
+  /**
+   * Verifies the signature on a given message
+   *
+   * @param {string} signedMessage the signed message for the signature
+   * @param {string} signature the signature to verify
+   * @param {string} address the address of the signer
+   * @returns {boolean} whether the signature is valid or not
+   */
+  verifySignature(signedMessage: string, signature: string, address: string): boolean {
```

Comment:
@rliberal-simtlix I did this, should i change type to UInt8Array?

Reply:
Looks good to me. Thank you.